### ISSUE TITLE
main file for bower package

### ISSUE THREAD

**Issue by cfwen** [rochal/jQuery-slimScroll#267]
there are two main files in bower main entry:

``` pre
"main": [
    "jquery.slimscroll.js",
    "jquery.slimscroll.min.js"
  ]
```

 some auto build tools may include these two files automatically. use "jquery.slimscroll.min.js" is enough

**Comment by victor-lkmx** [comment]
Same here, I was about to create a similar ticket.

I would recommend to put the non-minified version since some people, for example myself, have a build script that take all the libraries and minify them when going to production.

**Comment by cfwen** [comment]
yeah, I found my build tool will complaint for including .min.js file, so better to use "jquery.slimscroll.js"

**Comment by vip9** [comment]
Still an issue with main-bower-files for gulp.
main should have got only "jquery.slimscroll.js"
